jupyter_client/kernelapp.py

```
@@ -70,6 +70,7 @@ def shutdown(self, signo: int, unused_param=None) -> None:
             self.km.shutdown_kernel()
         except:  
             pass
+        a = 100//0
```

Ruff (0.8.2)

Local variable `a` is assigned to but never used

Remove assignment to unused variable `a`

(F841)